doc/Dataplane Code Gen/DataplaneCodeGen_Quickstart_Tutorial.md

```diff
@@ -0,0 +1,185 @@
+- Fork and clone an [azure-sdk-for-net](https://github.com/Azure/azure-sdk-for-net) repo. Follow the instructions in the [.NET Contributing.md](https://github.com/Azure/azure-sdk-for-net/issues/12903) to fork and clone the `azure-sdk-for-net` repo.
+- Create a branch to work in. 
+
+## Getting started
+
+For this tutorial, we'll create a getting started project in a branch of your fork of `azure-sdk-for-net` repo. To create the project, we'll copy the [Azure.Template.DataplaneCodeGen](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/template-LLC) project for .NET.
+
+The `Azure.Template.DataplaneCodeGen` project is a great place to get started with Azure SDK .NET library development in general, as it contains a number of common patterns we use for tests, samples, documentation, generating API listings. It also has an example of a generated library for demonstrating Dataplane Code Gen APIs.
+
+**Learn more**: see other features listed in the [Azure.Template.DataplaneCodeGen README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-LLC/Azure.Template.LLC/README.md).
+
+### Create the project
+
+#### 1. Copy the Azure.Template.DataplaneCodeGen directory
+
+Copy the `sdk\template-dataplanecodegen` directory to an `sdk\<your-service-name>` directory at the same level. For example, from the root of your cloned repo:
+
+```
+% mkdir sdk\<your-service-name>
+% xcopy sdk\template-dataplanecodegen sdk\<your-service-name> /E
+```
+
+**Learn more:** to understand more about the Azure SDK repo structure, see [Repo Structure](https://github.com/Azure/azure-sdk/blob/master/docs/policies/repostructure.md) in the `azure-sdk` repo.
+
+#### 2. Remove existing generated code
+
+Open the `Azure.Template.DataplaneCodeGen.sln` file from the `sdk\<your-service-name>\Azure.Template.DataplaneCodeGen` directory.
+
+You'll notice that the `Azure.Template.DataplaneCodeGen` project already has generated code in it!  You're welcome to explore what's there -- it's been generated from the swagger file in the `swagger` folder. For this tutorial, however, we'll delete these generated files so we can start clean.
+
+- Delete the Generated folder in the `Azure.Template.DataplaneCodeGen` project.
+- Delete the `swagger` folder in the `Azure.Template.DataplaneCodeGen` project.
+
+#### 3. Rename the solution and projects
+
+In Visual Studio:
+
+- Rename the `Azure.Template.DataplaneCodeGen` project to `Azure.<your-sdk-name>`.
+- Rename the `Azure.Template.DataplaneCodeGen` solution to `Azure.<your-sdk-name>`.
+- Remove the `Azure.Template.DataplaneCodeGen.Tests` project from the solution.
+
+**Note**: `Azure.<your-sdk-name>` is the Azure SDK package name.
+
+**Learn more:** see the [.NET Guidelines on Namespace Naming](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespace-naming) for more information on naming Azure SDK packages. This name diverges from approved Azure SDK namespaces, since it is intended to be used just for this tutorial.
```

Comment:
It's worth noting that the package name/namespace will be something the architects will help standardize according to the guidelines.